File: src/main/kotlin/com/geonyeong/insurance/entity/GuideContentEntity.java
package com.geonyeong.insurance.entity;

import com.geonyeong.insurance.dto.GuideContentId;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "guide_content")
@IdClass(GuideContentId.class)
@Getter
@Setter
public class GuideContentEntity{
    @Id
    private @Column(name = "guide_id") Long guideId;
    @Id
    private @Column(name = "guide_no") Long guideNo;
    @Id
    private @Column(name = "no") Long no;
    private @Column(name = "content") String content;
}

File: src/main/kotlin/com/geonyeong/insurance/entity/GuideContentEntity.kt
package com.geonyeong.insurance.entity

import com.geonyeong.insurance.dto.GuideIdDto
import javax.persistence.*

@Entity
@Table(name = "guide_content")
@IdClass(GuideIdDto::class)
data class GuideContentEntity(
        @Id
        @Column(name = "guide_id")
        var guideId:Long,
        @Id
        @Column(name = "guide_no")
        var guideNo:Long,
        @Id
        @Column(name = "no")
        var no:Long,
        @Column(name = "content")
        var content:String
){
        constructor():this(0,0,0,"")
}